contrib-docs/how-to-write/what-we-write.md

```diff
@@ -0,0 +1,91 @@
+---
+title: What we write
+slug: <MDN/Contribute/how-to-write/what-we-write>
+tags:
+  - <Guide>
+  - <Guidelines>
+  - Writing
+  - Documentation
+  - MDN Meta
+---
+<{{MDNSidebar}}>
+
+In this article, you'll find information about whether or not a given topic and/or type of content should be included on MDN.
+
+> **Note:** Be aware that all contributions to MDN fall under specific open source licenses; these are [described in detail](/en-US/docs/MDN/About#copyrights_and_licenses) on our [About MDN](/en-US/docs/MDN/About) page. Keep in mind that Mozilla's [Websites & Communications Terms of Use](https://www.mozilla.org/en-US/about/legal/terms/mozilla/) are in effect when you use or contribute to MDN. <!--- links need to be revisited--->
```

Comment:
Yeh super cool - we have license etc on the list, so we can go through towards the end of writing & check 👍